tasks/section_6_System_Maintenance.yaml

```diff
@@ -612,15 +612,14 @@
 # 6.2.17 Ensure shadow group is empty
 - name: 6.2.17 Ensure shadow group is empty
   block:
-    - name: 6.2.17 Ensure shadow group is empty | list
+    # Ensures there no user exists within the shadow group
+    - name: 6.2.17 Ensure shadow group is empty | checks /etc/group
+      command: sed -i '/^shadow:/s/:[^:]*$/:/' /etc/group
+    # Checks if there are users that have the shadow group assigned as their primary group (as this info won't appear in the /etc/group)
+    # If such users exist, the command will reassign their primary group to the one sharing the same name as their username
```

Comment:
does the "user" group always exist?

Reply:
It should. 
Note that it is not the group "user" the one that will be assigned, but the group with the same name as the user.

Every must have at least one primary group, it is common practice in most UNIXes and Linux distros, when user xyz is created, to automatically create a group named xyz and assign to it user xyz as its (only) member. 

- https://www.linuxquestions.org/questions/linux-newbie-8/why-is-the-group-name-and-user-name-the-same-913180/
https://unix.stackexchange.com/questions/377672/can-group-name-of-a-user-and-the-username-of-some-other-user-be-the-same
- https://www.linuxquestions.org/questions/linux-newbie-8/why-is-the-group-name-and-user-name-the-same-913180/
https://unix.stackexchange.com/questions/377672/can-group-name-of-a-user-and-the-username-of-some-other-user-be-the-same

Reply:
Yeah, I understood that it's the $USER group. :-)

No further comments here.